src/Content/Embed/EmbedVideo.tsx

```diff
@@ -37,11 +37,13 @@ function ThumbnailWrapper({
       onClick={() => setHideThumbnail(true)}
       onError={() => setError(true)}
       style={{
-        width: adjustedWidth,
-        height: adjustedHeight,
-        backgroundImage: `url(${error ? discordImageFailure : thumbnail})`,
+        width: "100%",
```

Comment:
backgroundImage has been removed?

Reply:
Instead, I move the image to the image tag to make it responsive and fit on the small screen without the edges being cut.